src/index.html

```diff
@@ -17,11 +17,349 @@
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo-MOYO"
+          class="logo__img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="wrapper">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="../src/images/imac.jpeg"
+          alt="зображення товару"
+          class="card__background_image"
+        />
+        <p class="card__text">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product_code">Product code: 195434</p>
+        <div class="starsBox">
+          <div class="starsBox__container">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+          </div>
+          <p class="reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price_list">
+          <p class="card__price">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+        <button
+          class="card__buy_button"
+          data-qa="card-hover"
+        >
+          BUY
+        </button>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```